WebInstall/explainer_cross_domain.md

```diff
@@ -190,9 +196,20 @@ A new web-manifest boolean key `allow_all_install_sources` signals that the appl
 }
 ```
 
-This new manifest field will protect the app from being listed in undesirable repositories and give the developer absolute control about where do they want the PWA to be installed from. At best, the developer can allow the PWA to be installed from any site ("`*`"), at its most restrictive, it can only allow installing from the app's same scope. This field is only for the JS API and does not interfere with existing ways of installing PWAs through mechanisms like enterprise policies.
+##### Controlling installations from any origin
+To ensure that the developer can have control of where their app can be installed from independent of the implementors default behaviour, a new web-manifest boolean key `allow_all_install_sources` can tell the UA that the application can be installed from any or no other origin.
+
+* if set to `true`, the `install_sources` list is ignored (if included) and the app can be installed from any origin.
+* if set to `false`, the `install_sources` list is ignored (if included) and the app can be installed only from the same-origin it is hosted.
```

Comment:
What about the case where the browser allows install by default, but the developer only wants to allow a few vetted stores to install? This would block that scenario.

What about:
```suggestion
* if set to `false`, the `install_sources` list is respected (if included) and the app can be installed only from the origins listed in `install_sources` or the same-origin it is hosted.
```

Reply:
Chatted offline. Diego's suggestion was to change this to discuss the order of presidence: 
1. `install_sources` (if provided the UA assumes the dev wants to block by default)
2. `allow_all_install_sources`
3. UA default behavior of allow or block by default.